addons/sys_core/stringtable.xml

```diff
@@ -380,6 +381,7 @@
             <Chinese>天線拉直</Chinese>
             <French>Antenne redressée</French>
             <Portuguese>Antena endireitada</Portuguese>
+            <Spanish>Antena derecha</Spanish>
```

Comment:
```suggestion
            <Spanish>Antena enderezada</Spanish>
```

Reply:
"Antena enderezada" is a literal translation and isn't correct, a more correct one is "Enderezar antena" i think, that mean´s put the antenna straight